app/models/course/condition/assessment.rb

```diff
@@ -0,0 +1,34 @@
+class Course::Condition::Assessment < ActiveRecord::Base
```

Comment:
I don't think so. We use the same convention for `Course::Condition::Achievement`.